web/src/store/index.js

```diff
@@ -39,17 +41,39 @@ const plotDefaults = {
   pca: {
     data: null,
     valid: false,
+    loading: false,
+    args: {},
+    type: plot_types.TRANSFORM,
   },
   loadings: {
     data: null,
     valid: false,
+    loading: false,
+    args: {},
+    type: plot_types.TRANSFORM,
   },
 };
 
+analyses.forEach(({
+  args,
+  path,
+  key,
+  type,
+}) => {
+  plotDefaults[key || path] = {
```

Comment:
Why is there a distinction between `key` and `path`? It seems like it would be better to be explicit about this in the `analyses` list, rather than put the mysterious comment there (which I think I now understand :smiley_cat:) and this slippery-feeling logic here.

Reply:
cc @sgratzl

Reply:
in the end the `path` has to match the `plotName` (formally known as `key`) used when defining the mixin. We could separate the two or leave it as convention

Reply:
Because `key` is no longer a property, I'm just going to change this line to use `path`.